README.md

```diff
@@ -1,24 +1,23 @@
-[![DOI](https://zenodo.org/badge/20756/DDTBOX/DDTBOX.svg)](https://zenodo.org/badge/latestdoi/20756/DDTBOX/DDTBOX)
 # DDTBOX
 Decision Decoding Toolbox
 
 
-Copyright (c) 2013--2016 Stefan Bode and contributors.
+Copyright (c) 2013--2017 Stefan Bode and contributors.
 
 Unless otherwise specified, code is distributed under the GNU Public License (GPL) version 2, and documentation under a Creative Commons Attribution-Share-Alike 4.0 International license.
 
 <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
 
 We hope that you find the software and documentation useful.
 If you publish an analysis using the toolbox, we ask that you cite us. 
-For now, please cite us via GitHub, but we are currently preparing on a publication to submit to peer review. 
-A sample citation would be:
+We are currently preparing on a publication to submit to peer review. 
+For now, please cite our BioRxiv preprint:
 
-Bode, S., Bennett, D., Feuerriegel, D., & Alday, P. (2016). The Decision Decoding Toolbox -- a multivariate pattern analysis toolbox for event-related potentials. DOI: 10.5281/zenodo.48143
+Bode, S., Feuerriegel, D., Bennett, D., & Alday, P.M. (2017). The Decision Decoding ToolBOX (DDTBOX) -- A multivariate pattern analysis toolbox for event-related potentials. DOI: 10.1101/153189 
```

Comment:
I think they should still cite version -- if using the an official release, cite the Zenodo DOI; if using current master or a development branch, then the commit hash. This helps with reproducibility.